larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -193,7 +208,7 @@ class BaseBConv2DOpModel : public SingleOpModel {
   BaseBConv2DOpModel(register_function registration, const TensorData& input,
                      const TensorData& filter, const TensorData& output,
                      int stride_width = 1, int stride_height = 1,
-                     enum Padding padding = Padding_VALID,
+                     const char* padding_str = "VALID",
```

Comment:
I would rather pass an `enum` as before and when I need a string of enum use the utility function above to get its string value.